pkg/cache/types/snapshot.go

```diff
@@ -21,6 +21,12 @@ type SnapshotResource struct {
 	Serialized *anypb.Any
 	// Optional
 	Version string
+
+	// Optional - marks resource as on-demand only (e.g. for OdCDS).
+	// Only supported for the snapshot cache.
```

Comment:
```suggestion
```

Reply:
In theory this could be added to the linear cache, and this comment would likely be outdated